services/web/server/src/simcore_service_webserver/login/_registration_api.py

```diff
@@ -95,3 +100,19 @@ async def send_account_request_email_to_support(
             template_name,
             f"{support_email=}",
         )
+
+
+async def generate_captcha() -> tuple[str, bytes]:
+    captcha_text = "".join(random.choices(string.digits, k=6))
```

Comment:
To generate codes always use functions in https://github.com/itisfoundation/osparc-simcore/blob/21ea75e6d343af777d7dc247e1ab25b66bbbec67/packages/service-library/src/servicelib/utils_secrets.py